DigitalPassaport/DigitalPassaport/Infactuture/Resources/NetworkRouter.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  NetworkRouter.swift
+//  DigitalPassaport
+//
+//  Created by Milena Predic on 29.8.23..
+//
+
+import Alamofire
+
+protocol NetworkRoutable {
+    var method: HTTPMethod { get }
+    var path: String { get }
+}
+
+enum Router: NetworkRoutable {
```

Comment:
The name should be for example AccountRouter or PassesRoute, or something else. But the point is that in one moment we will have more than one service, and each service should have some network router that inherit NetworkRoutable protocol. 
Naming Router more means something related to the UI